packages/driver/src/cy/actionability.ts

```diff
@@ -8,7 +8,6 @@ import $utils from './../cypress/utils'
 import type { ElWindowPostion, ElViewportPostion, ElementPositioning } from '../dom/coordinates'
```

Comment:
This file's changes involve undoing https://github.com/cypress-io/cypress/pull/27860